template/src/project_name/main/views.py

```diff
@@ -31,3 +31,14 @@ def get(
                 "revision": settings.GIT_COMMIT_COUNT,
             }
         )
+
+
+class TriggerError(generic.View):
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏Should be removed

Whilst useful for local testing initially I don't think this should be a standard part of the template. Accidentally leaving in place on a real site would open us to trouble s anything could trigger errors remotely over and over.

Reply:
I see, will remove.